test/e2e/argo_server_test.go

```diff
@@ -254,14 +244,31 @@ func (s *ArgoServerSuite) TestPermission() {
 			Status(403)
 	})
 
-	// Test list workflows in good ns with forbidden ns token
-	s.bearerToken = forbiddenToken
-	s.Run("List", func(t *testing.T) {
-		s.bearerToken = forbiddenToken
+	// Test list workflows with bad token
+	s.bearerToken = badToken
+	s.Run("List workflows with bad token", func(t *testing.T) {
 		s.e(t).GET("/api/v1/workflows/" + nsName).
 			Expect().
 			Status(403)
 	})
+
+	// TODO: Test list archived wf with good token after archived wf APIs changes are finalized.
```

Comment:
TODO?

Reply:
Have you finished adding `namespace` in listing archived wf URL? @alexec